install/kubernetes/cilium/values.yaml

```diff
@@ -515,3 +509,50 @@ global:
 
   cnpStatusUpdates:
     enabled: false
+
+# Configures the hubble-cli subchart
+hubble-cli:
```

Comment:
what do think about moving these two blocks all the way up, right after _preflight_ subchart values? I think it's easier for user to follow.